classes/03class/exercises/c03-docker06/drii-cavalcanti/c03-docker06.txt

```diff
@@ -0,0 +1,17 @@
+$ sudo docker build -t hello-name:v1 --build-arg myname=Adriana .
+    Sending build context to Docker daemon  4.096kB
+    Step 1/3 : FROM alpine
+    ---> e7d92cdc71fe
+    Step 2/3 : ARG myname=what_is_your_name
+    ---> Running in c2ac29f6198a
+    Removing intermediate container c2ac29f6198a
+    ---> 64ce130dbc9a
+    Step 3/3 : RUN echo "Hello $myname"
+    ---> Running in ce80b815547d
+    Hello Adriana
+    Removing intermediate container ce80b815547d
+    ---> 0a3338a3c5e0
+    Successfully built 0a3338a3c5e0
+    Successfully tagged hello-name:v1
+$ sudo docker run -t hello-name:v1
+/ # 
```

Comment:
Hi Adriana.. that's not quite right..
The objective is to have the value passed to the container during execution.. Your container image should do something like this:
```
$ sudo docker run hello-name:v1 Adriana
Hello Adriana
$
```